Common/src/main/scala/nl/lumc/sasc/biopet/test/SummaryPipeline.scala

```diff
@@ -238,6 +242,39 @@ trait SummaryPipeline extends PipelineSuccess {
     val exesDb = Await.result(summaryDb.getExecutables(runId = Some(runId), toolName = Some(exe)), Duration.Inf)
     exesDb shouldBe empty
   }
+
+  @DataProvider(name = "moduleSchemas")
+  def moduleSchemasProvider() = {
+    val modulesUsed = Await.result(summaryDb.getModules(runId = Some(this.runId)), Duration.Inf)
```

Comment:
This does work but what we don't test here is things are there or not there if we are expecting them. We rely on the module table in the database. 

For now this can stay like this because changing this that it checks if modules exist or not based on the settings is more work.